tle/cogs/hard75Challenge.py

```diff
@@ -169,7 +114,7 @@ async def streak(self,ctx):
         user_id=ctx.author.id
         res=cf_common.user_db.get_hard75_status(user_id)
         if res is None:
-            raise Hard75CogError('There is no record of yours in the DB have you ever completed a Hard75 problem?')
+            raise Hard75CogError('You haven\'t completed a Hard75 problem? Get started with ;hard75 letsgo')
```

Comment:
maybe something like - "seems like you haven't completed any hard75 problem! Get started with ;hard75 letsgo"

Reply:
Seems sound like we dont know what we are doing. We **know** he hasnt done any problem yet.

Reply:
But the ? is wrong. i fix it in next version

Reply:
sure works!